src/TestFramework/DataFaker.php

```diff
@@ -1076,4 +1076,67 @@ public function riskScore()
     {
         return $this->intBetween(-2, 100);
     }
+
+    /**
+     * Return a payment instrument name.
+     *
+     * @return string
+     */
+    public function paymentInstrumentName()
+    {
+        return $this->randomCharacters(self::ALPHABET_UPPER . self::DIGITS, $this->intBetween(4, 10));
+    }
+
+    /**
+     * Return a payment network.
+     *
+     * @return string
+     */
+    public function paymentNetwork()
+    {
+        return $this->randomCharacters(self::ALPHABET_UPPER . self::ALPHABET_LOWER, $this->intBetween(4, 16));
```

Comment:
Does it make sense to randomly picking from array ['Visa', 'MasterCard', 'Amex']. So it's more inline with what we expect from Apple